bin/comment_count.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
As a note, I'd recommend writing this file in python (where use `subprocess.run` around `cloc --csv --quiet`, and then rest of code is just python) so that:
1. it's more portable (I cannot run this on my macOS terminal as it doesn't have `mapfile` function, which is a bash 4+ builtin and mac comes with zsh (default) and bash 3)
2. it's simpler for new students / programmers to edit and doesn't require futzing with bash's weird string handling.